tests/util_tests/test_user_settings.py

```diff
@@ -11,95 +11,98 @@
 from __future__ import with_statement
 
 import os
-import unittest2 as unittest
+import uuid
+
+from tank_test.tank_test_base import TankTestBase
+from tank_test.tank_test_base import setUpModule # noqa
+
 from mock import patch
 
 from tank.util import EnvironmentVariableFileLookupError
 from tank.util.user_settings import UserSettings
+from sgtk import TankError
 
 
-class MockConfigParser(object):
+class UserSettingsTests(TankTestBase):
     """
-    Mocks the config parser object used internally by the UserSettings class.
```

Comment:
Refactored the unit tests so we weren't mocking code anymore and relied on actual files on disk. I feel this make the test easier to understand.